errext/exitcodes/codes.go

```diff
@@ -20,4 +20,5 @@ const (
 	CannotStartRESTAPI       ExitCode = 106
 	ScriptException          ExitCode = 107
 	ScriptAborted            ExitCode = 108
+	Panic                    ExitCode = 109
```

Comment:
Just to be clear, since the others are very high-level and functional reasons.

```suggestion
	GoPanic                    ExitCode = 109
```

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/cf4fd49f36b32dabf34f5474ffcfeec953758c32..e4265bbdf8598a4887a5414b9f62a1719fb17a99)